src/pocketmine/plugin/PluginManager.php

```diff
@@ -532,22 +532,25 @@ public function registerEvents(Listener $listener, Plugin $plugin) : void{
 					throw new PluginException("Event handler " . Utils::getNiceClosureName($handlerClosure) . "() declares invalid/unknown priority \"" . $tags["priority"] . "\"");
 				}
 
-				$ignoreCancelled = false;
 				if(isset($tags["ignoreCancelled"])){
-					switch(strtolower($tags["ignoreCancelled"])){
+					throw new PluginException("Event handler " . Utils::getNiceClosureName($handlerClosure) . "() declares obsolete tag @ignoreCancelled");
```

Comment:
This doesn't make a whole lot of sense. The new behaviour is to "ignore cancelled" anyway, so this is just a patchy way to warn of a behavioural change that didn't have any effect in this case anyway.

Reply:
This is like a shouty warning to let people notice about this change, so although it usually targets victims that are not affected by the new behavioiur, it is still ok.

Reply:
It's pointless since it will only shout at people who aren't affected.